src/globals.h

```diff
@@ -37,23 +37,11 @@ extern uint8_t G_io_seproxyhal_spi_buffer[IO_SEPROXYHAL_BUFFER_SIZE_B];
 #define MAX_PARTNER_NAME_LENGHT 15
 
 #define PARTNER_NAME_PREFIX_FOR_FUND "To "
-#define PARTNER_NAME_PREFIX_SIZE     (sizeof(PARTNER_NAME_PREFIX_FOR_FUND) - 1)  // Remove trailing '\0'
 
-#pragma pack(push, 1)
 typedef struct partner_data_s {
-    uint8_t name_length;
```

Comment:
Do you think this change should be part of this PR? Also,  this seems to affect the original app-exchange and not related to tezos. Better to suggest these improvements separately and not as part of tezos upgrade.